### PR TITLE
415 engine not running warning

### PR DESCRIPTION
**Pull request by ricardovdheijden**
#415 

- Shows a toast message when the action cannot be performed.
- When the onClick happens a refetch is triggered in the useEngineStatus hook.
- As a bonus: the engine status at the top navigation will update too

### CODE REVIEW

File: packages/orchestrator-ui-components/src/messages/en-US.json [hunk 1]
```diff
@@ -37,6 +37,8 @@
         "start_workflow_title": "Start workflow"
     },
     "errors": {
+        "notAllowedWhenEngineIsNotRunningMessage": "This action is not allowed when the Workflow Engine is not running",
+        "notAllowedWhenEngineIsNotRunningTitle": "Workflow Engine is not running",
         "retrieve_stored_settings": "Something went wrong restoring your settings. Falling back to default settings",
```
Comment (acidjunk) [inline packages/orchestrator-ui-components/src/messages/en-US.json:42]: the mixing of camel case + snake case looks a bit weird.

Partially caused by form field names: that are python attributes on pydantic classes: so snake cased.

File: packages/orchestrator-ui-components/src/messages/nl-NL.json [hunk 2]
```diff
@@ -37,6 +37,8 @@
         "start_workflow_title": "Start workflow"
     },
     "errors": {
+        "notAllowedWhenEngineIsNotRunningMessage": "This action is not allowed when the Workflow Engine is not running",
```
Comment (acidjunk) [inline packages/orchestrator-ui-components/src/messages/nl-NL.json:40]: translate to dutch?

**Review by acidjunk — APPROVED** [review]

File: packages/orchestrator-ui-components/src/messages/nl-NL.json (hunk 2 above)
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/messages/nl-NL.json]: done

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/WfoStartCreateWorkflowButtonComboBox.tsx [hunk 3]
```diff
@@ -40,7 +48,19 @@ export const WfoStartCreateWorkflowButtonComboBox = () => {
             )
             .sort((a, b) => a.label.localeCompare(b.label)) ?? [];
 
-    const handleOptionChange = (selectedProduct: WorkflowComboBoxOption) => {
+    const handleOptionChange = async (
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/WfoStartCreateWorkflowButtonComboBox.tsx:51]: Part of this code seems to repeat a lot. Could it be abstracted into a useCheckEngineRunning hook or something?

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/WfoStartCreateWorkflowButtonComboBox.tsx (hunk 3 above)
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/WfoStartCreateWorkflowButtonComboBox.tsx:44]: Done, introduced new hook -- moved logic there, including the toast

**Pull request merged**